app/javascript/packages/session/index.ts

```diff
@@ -0,0 +1,32 @@
+import { request } from '@18f/identity-request';
+
+export interface SessionStatusResponse {
+  /**
+   * Whether the session is still active.
+   */
+  live: boolean;
```

Comment:
WDYT of `isLive`? or is this supposed to map directly to the JSON body
```suggestion
  isLive: boolean;
```

Reply:
It doesn't necessarily need to map directly to the JSON body, though having them aligned does help simplify things. Generally I am in favor of prefixes like that for boolean variables in JavaScript. Maybe some light re-mapping is appropriate here, since the package serves as an abstraction over the endpoint, so it should hopefully be the only place in the code dealing with that detail.

Reply:
Implemented response mapping in c6d7c7f5e.